dss-api

```diff
@@ -1,9 +1,31 @@
 #!/usr/bin/env python
 
+"""
+Entry point for starting a local test HCA DSS API server.
+"""
+
+import os
+import sys
 import logging
+import argparse
+
+from chalice.cli import CLIFactory, run_local_server
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("--port", type=int, default=5000)
+parser.add_argument("--no-debug", dest="debug", action="store_false")
+parser.add_argument("--project-dir", help=argparse.SUPPRESS,
+                    default=os.path.join(os.path.dirname(__file__), "chalice"))
+parser.add_argument("--log-level",
```

Comment:
i have this vague notion that there's different logging contexts.  is that something we want to have manipulated at the command-line level?